src/NuGet.Clients/NuGet.CommandLine/Commands/ClearCacheCommand.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using NuGet.Configuration;
+
+namespace NuGet.CommandLine.Commands
+{
+    [Command(typeof(NuGetCommand), "clearcache", "ClearCacheCommandDescription", MaxArgs = 0,
+            UsageSummaryResourceName = "ClearCacheCommandSummary", UsageExampleResourceName = "ClearCacheCommandExamples")]
+    public class ClearCacheCommand
+        : Command
+    {
+        [Option(typeof(NuGetCommand), "ClearCacheCommandClearGlobalPackagesDescription")]
+        public bool ClearGlobalPackages { get; set; }
+
+        public override Task ExecuteCommandAsync()
+        {
+            // Clear the NuGet machine cache
+            if (!string.IsNullOrEmpty(MachineCache.Default?.Source))
+            {
+                Console.WriteLine($"{LocalizedResourceManager.GetString("ClearCacheCommand_ClearingNuGetCache")}: {MachineCache.Default.Source}");
```

Comment:
Most strings have a '{0}' and use LocalizedResourceManager to fill in the parameter instead of concatenating the strings like this is doing. I suggest using that instead to keep things consistent and clear.